base/common/src/test/java/com/netscape/cms/servlet/test/ConfigurationTest.java

```diff
@@ -52,306 +40,13 @@
 import org.mozilla.jss.pkix.crmf.ProofOfPossession;
 import org.mozilla.jss.pkix.primitive.Name;
 import org.mozilla.jss.pkix.primitive.SubjectPublicKeyInfo;
-import org.mozilla.jss.util.Password;
-
-import com.netscape.certsrv.system.SystemCertData;
-import com.netscape.cmsutil.crypto.CryptoUtil;
 
 /**
  * @author alee
  *
  */
 public class ConfigurationTest {
```

Comment:
As all the code has been removed but for one static method, this class can become an interface with a default method. This would resolve the "hide the implicit constructor" warning. It should then be renamed to something more suitable to it's new role, like `CRMFRequestGenerator`.